packages/components/codemods/Well/__snapshots__/transformWell.spec.ts.snap

```diff
@@ -0,0 +1,66 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`updateFileContents should not update Well components imported from @kaizen/draft-well or legacy packages 1`] = `
+"import * as React from "react";
+// @ts-ignore
+import { Well } from "@kaizen/draft-well";
+export const TestWellComponentUnhappy = (): JSX.Element => (<div>
+    <Well variant="default">Test 1 should have default variant</Well>
+    <Well>Test 2 should have no change</Well>
+    <Well>Test 3 should have informative variant</Well>
```

Comment:
This appears to have failed (assuming your text is still the expectation)

Reply:
Weird, TestWellComponentUnhappy doesn't exist anymore, would have though updateSnapshots would have removed it 🤷